docs/subnets/create-a-subnet.md

```diff
@@ -11,31 +11,31 @@ You **do not** have to create a subnet to mine or validate on the Bittensor netw
 ## Recommended flow for deploying your subnet 
 
 Before you deploy your first subnet on the mainchain, we strongly recommend that you follow the below order:
-1. **Local first**: First create a subnet on your local, and develop and test your incentive mechanism on the local subnet. 
-2. **Testchain next**: After you are satisfied with it, next create a subnet on the Bittensor testchain, and test and debug your incentive mechanism on this testchain subnet. 
-3. **Mainchain last**: Finally, only after you completed the above steps, create a subnet on the Bittensor mainchain. 
+1. **Local first**: Create a subnet on your local, and develop and test your incentive mechanism on the local subnet.
+2. **Testchain next**: After you are satisfied with it, create a subnet on the Bittensor testchain, and test and debug your incentive mechanism on this testchain subnet.
+3. **Mainchain last**: After completing the above steps, create a subnet on the Bittensor mainchain.
 
 :::tip Subnet Creation Limits
 Subnet creations are limited to **one subnet creation per 7200 blocks** (approximately one per day).
 :::
 
 ## Prerequisites
 
-To create a subnet, whether locally or on testchain or on mainchain, make sure that:
+To create a subnet, whether locally, on testchain, or on mainchain, make sure that:
 
 - You [installed Bittensor](../getting-started/installation.md). 
 - You have already [created a wallet or know how to create one](../getting-started/wallets.md#creating-a-local-wallet). 
 
 :::tip Registering vs creating vs purchasing a slot 
-In Bittensor we say "registering your keys in a subnet" to mean purchasing a UID slot in the subnet and you will then either validate or mine on this UID. This step is also referred as **purchasing a slot**. On the other hand, "creating a subnet" will create a subnet and give you its `netuid`.
+In Bittensor, we say "registering your keys in a subnet" means purchasing a UID slot in the subnet, and you will then either validate or mine on this UID. This step is also referred to as **purchasing a slot**. On the other hand, "creating a subnet" will create a subnet and give you its `netuid`.
```

Comment:
It reads better in either way below (I prefer the second way):
- In Bittensor, we say ... to mean purchasing ... (same as original but with a comma after Bittensor)
- In Bittensor, "registering your keys in a subnet" means purchasing ...

Reply:
I think you meant to write "In Bittensor, we say "registering your keys in a subnet" to mean purchasing a UID slot in the subnet, and you ..." (sorry I was not clear in my first way above).